.github/workflows/msrv-windows-result.yml

```diff
@@ -29,13 +29,3 @@ jobs:
         run: cargo check -p windows-result --all-features
       - name: Check Default Features
         run: cargo check -p windows-result
-      - name: Check Slim Errors
```

Comment:
This yml is for testing MSRV and nothing more. The `windows_slim_errors` testing is provided by a dedicated slim_errors.yml just like how `windows_raw_dylib` is tested.